src/Sylius/Bundle/InstallerBundle/Requirement/SettingsRequirements.php

```diff
@@ -16,99 +16,45 @@
 
 class SettingsRequirements extends RequirementCollection
 {
-    const REQUIRED_PHP_VERSION = '5.5.9';
-    const RECOMMENDED_PHP_VERSION = '5.6.13';
+    const RECOMMENDED_PHP_VERSION = '7.0';
 
     public function __construct(TranslatorInterface $translator)
     {
-        parent::__construct($translator->trans('sylius.settings', [], 'requirements'));
-
-        $on = $translator->trans('sylius.settings.on', [], 'requirements');
-        $off = $translator->trans('sylius.settings.off', [], 'requirements');
+        parent::__construct($translator->trans('sylius.settings.header', [], 'requirements'));
 
         $this
             ->add(new Requirement(
-                $translator->trans('sylius.settings.version', [], 'requirements'),
-                version_compare(phpversion(), self::REQUIRED_PHP_VERSION, '>='),
```

Comment:
I thought it will be handled by Composer and forgot to revert it after @pjedrzejewski's comment.